packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -3789,7 +3787,7 @@ curl -v -X GET \
 }
 ```
 
->**Note:** The `type` element and the `schema` and `type` links are present only if the User Types feature is enabled. If enabled, they appear for all users, even those with the default User Type. See <ApiLifecycle access="ea" /> [User Types](/docs/reference/api/user-types).
+>**Note:** The `type` element and the `schema` and `type` links are present for all users, even those with the default User Type. See [User Types](/docs/reference/api/user-types).
```

Comment:
Same question as above. In my opinion we might not need this anymore.

Reply:
Yeah, I agree with the previous comment that this may as well come out.